docs/source/quickstart.mdx

```diff
@@ -1,211 +1,285 @@
-# Quick Start
-
-The quick start is intended for developers who are ready to dive in to the code, and see an end-to-end example of how they can integrate 🤗 Datasets into their model training workflow. For beginners who are looking for a gentler introduction, we recommend you begin with the [tutorials](./tutorial).
-
-In the quick start, you will walkthrough all the steps to fine-tune [BERT](https://huggingface.co/bert-base-cased) on a paraphrase classification task. Depending on the specific dataset you use, these steps may vary, but the general steps of how to load a dataset and process it are the same.
+# Quickstart
+
+This quickstart is intended for developers ready to dive into the code and see an end-to-end example of how to integrate 🤗 Datasets into their model training workflow. If you are a beginner or a non-developer, we recommend starting with our [tutorials](./tutorial), where you will get a more thorough introduction.
+
+Each dataset is unique, and depending on the task, some datasets may require additional steps to prepare the data. But you can always use 🤗 Datasets tools to load and process a dataset. Choose the type of dataset you want to work with, and let's get started!
+
+<div class="mt-4">
+   <div class="w-full flex flex-col space-y-4 md:space-y-0 md:grid md:grid-cols-3 md:gap-y-4 md:gap-x-5">
+      <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#audio"
+         ><div class="w-full text-center bg-gradient-to-r from-violet-300 via-sky-400 to-green-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Audio</div>
```

Comment:
`via-sky-400` is still present here (which is not in Tailwind v2). Is it expected?

Reply:
Yeah, I'd like to keep that if possible!